feature-libs/organization/order-approval/components/details/order-approval-details.module.ts

```diff
@@ -18,6 +18,7 @@ import {
   OrderDetailTotalsComponent,
   SpinnerModule,
 } from '@spartacus/storefront';
+import { ApproverGuard } from '../../core/public_api';
```

Comment:
Good catch!

That import was wrong.  However we import the guard using a relative path because the approvals feature does not have a standalone "core" entry point.  Just root, asstes, and another one for the rest.